velodyne_pointcloud/src/conversions/transform.cc

```diff
@@ -43,11 +43,11 @@ namespace velodyne_pointcloud
     srv_->setCallback (f);
     
     // subscribe to VelodyneScan packets using transform filter
-    velodyne_scan_.subscribe(node, "velodyne_packets", 10);
+    velodyne_scan_.subscribe(node, "velodyne_packets", 15);
```

Comment:
Is there a particular reason/rationale behind this change?

Reply:
I'll have to ask around about this one. This PR was before I started working with the team. The 100 seems excessive.